### DISCUSSION TITLE
Commits are all verified but says merging is blocked

### DISCUSSION THREAD

**Discussion by kalavipin** [community/community#35251] [Enterprise]
Branch settings for main include - signed commits, PR review and 1 approval. But even after approvals and verified signatures, Ui shows merging is blocked. It woks from command line. Please help

**Comment by yixi8524** [comment]
Same issue in our Enterprise (3.6.4), Testing with new repo, branch protection only opens the `Require signed commits.` Only one signed commit in this PR, UI always show `The base branch requires all commits to be signed`

![image](https://user-images.githubusercontent.com/9018127/212581536-f7118838-223a-4384-9467-2e67a0a83afc.png)

**Comment by rodrigofitti** [comment]
Same issue here in GitHub Enterprise Server 3.9.10, with the same message "The base branch requires all commits to be signed."

**Comment by FlorentP42** [comment]
Quick link to a [stackoverflow issue](https://stackoverflow.com/questions/75854360/github-merging-is-blocked-because-all-commits-have-to-be-signed-but-they-are-si) reporting the exact same problem on public GitHub (not even Enterprise) and sadly not getting much attention either despite being open for more than one year...

**Comment by suzuki-shunsuke** [comment]
This discussion is categorized at `Enterprise` and the label `Enterprise` is given, but the problem occurs on public GitHub too.

- e.g. https://github.com/suzuki-shunsuke/tfaction/pull/1989

<img width="971" alt="image" src="https://github.com/user-attachments/assets/2bcbf72c-7183-4a7f-9f54-3fb32947c40b">

All commits were verified but the auto-merge got disabled.

> auto-merge was automatically disabled [yesterday](https://github.com/suzuki-shunsuke/tfaction/pull/1989#event-15113133774)
> Base branch requires signed commits

**Comment by JorenVsyntra** [comment]
Hey everyone!
